core/app/models/spree/payment.rb

```diff
@@ -145,6 +145,11 @@ def can_credit?
       credit_allowed > 0
     end
 
+    # @return [Boolean] true when this payment has been fully refunded
+    def fully_refunded?
+      refunds.sum(:amount) >= amount
```

Comment:
Fair enough. I don't like when we try to catch problems tangentially, but in it's only current use that case would raise an error, which would probably be useful. I'm cool with swicthing to `==`.